conf/application.conf

```diff
@@ -200,6 +200,20 @@ external-url {
   equality-commission-for-northen-ireland="https://www.equalityni.org/Home"
   digital-accessibility-centre="http://www.digitalaccessibilitycentre.org/"
   report-accessibility-problem="https://www.tax.service.gov.uk/contact/accessibility"
+  government-gateway-user-id-and-password="https://www.gov.uk/log-in-register-hmrc-online-services"
+  contact-hmrc="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/capital-gains-tax-enquiries-for-individuals-employees-and-self-employed"
+  report-client-cgt-on-uk-property-disposals="https://www.tax.service.gov.uk/capital-gains-tax-uk-property/manage-clients-capital-gains-tax-on-uk-property-account"
```

Comment:
this is one of our urls. If the url has `www.tax.service.gov.uk/capital-gains-tax-uk-property` in it that means it's ours. We shouldn't hard code our own url's - use the reverse routes to give you a url instead.

Reply:
Done.